src/rpc/server.cpp

```diff
@@ -322,6 +322,7 @@ void StopRPC()
 {
     LogPrint("rpc", "Stopping RPC\n");
     deadlineTimers.clear();
+    DeleteAuthCookie();
```

Comment:
I think this could/should be broken out into it's own PR. It's wasn't obvious right away why this was being added.